easybuild/easyconfigs/c/CDO/CDO-1.9.10-foss-2020b.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'ConfigureMake'
+
+name = 'CDO'
+version = '1.9.10'
+
+homepage = 'https://code.zmaw.de/projects/cdo'
+description = """CDO is a collection of command line Operators to manipulate and analyse Climate and NWP model Data."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
```

Comment:
@arielzn Toolchain can be "lowered" to `gompi`?

Reply:
it seems, indeed